site/content/en/references/kustomize/kustomization/patchesStrategicMerge/_index.md

```diff
@@ -7,6 +7,19 @@ description: >
     Patch resources using the strategic merge patch standard.
 ---
 
+{{% pageinfo color="warning" %}}
+The `patchesStrategicMerge` field was deprecated in v5.0.0. The `bases` field was deprecated in v2.1.0. This field will never be removed from the
```

Comment:
```suggestion
The `patchesStrategicMerge` field was deprecated in v5.0.0. This field will never be removed from the
```

Reply:
🤦‍♀️ Sorry about that, I should be proof reading my own PRs...